src/utils/isNextMonth.js

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment';
+
+export default function isNextMonth(a, b) {
+  if (!moment.isMoment(a) || !moment.isMoment(b)) return false;
```

Comment:
what non-moment values do you expect in these functions? I’d prefer a throw when it’s an unexpected value.

Reply:
I'll update this and the other spot, thanks.

Reply:
I think this is to allow one or both of a and b to be null (startDate and endDate are sometimes null)